pkg/execution/executor/executor.go

```diff
@@ -342,7 +358,7 @@ func (e *executor) Schedule(ctx context.Context, req execution.ScheduleRequest)
 		eventIDsStr = append(eventIDsStr, id.String())
 	}
 	spanID := telemetry.NewSpanID(ctx)
-	span.SetAttributes(attribute.StringSlice(consts.OtelSysEventIDs, eventIDsStr))
+	span.SetAttributes(attribute.String(consts.OtelSysEventIDs, strings.Join(eventIDsStr, ",")))
```

Comment:
I'm confused by this change - `StringSlice` was an issue because it serialized the value as a string, so instead we're just making a string?

Reply:
it'll serialize to this in the DB `'["<ULID>"]'`, and doesn't store it as an array.
it's pretty hard to convert it back to `Array(String)` from this format, and instead making it like `<ULID>,<ULID>` is a lot easier because we can do string splitting.

Reply:
it'll serialize to this in the DB `'["<ULID>"]'`, and doesn't store it as an array.
it's pretty hard to convert it back to `Array(String)` from this format, and instead making it like `<ULID>,<ULID>` is a lot easier because we can do string splitting.

Reply:
Discussed in person and agreed it's easier with , delimited string.